src/com/yetanalytics/lrs_admin_ui/subs.cljs

```diff
@@ -160,3 +161,133 @@
  :<- [:oidc/show-local-login?]
  (fn [show-local-login? _]
    show-local-login?))
+
+;; Status Dashboard
+(reg-sub
+ :status/enabled?
+ (fn [db _]
+   (::db/enable-admin-status db false)))
+
+(reg-sub
+ :db/status
+ (fn [db _]
+   (::db/status db)))
+
+(reg-sub
+ :status/params
+ :<- [:db/status]
+ (fn [status _]
+   (:params status)))
+
+(reg-sub
+ :status.params/timeline-unit
+ :<- [:status/params]
+ (fn [params _]
+   (:timeline-unit params "day")))
+
+(reg-sub
+ :status.params/timeline-since
+ :<- [:status/params]
+ (fn [params _]
+   (:timeline-since params (-> js/Date
+                               .now
+                               ;; two weeks ago
+                               (- 12096e5)
+                               (js/Date.)
+                               .toISOString))))
```

Comment:
This could probably go into your new util namespace.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/224b95f1c0b34ddb84a6ef5fc604480071c5b5bc

Reply:
Ah a nice and clear function name